ui/app/common/pnc-client/message-bus/messageBus.js

```diff
@@ -61,6 +61,14 @@
         }
       });
 
+      messageBus.onGenericSettingNewAnnouncement(notification => {
+        if (notification.message) {
+          let message = JSON.parse(notification.message);
+            $rootScope.$broadcast(events.NEW_ANNOUNCEMENT, message);
```

Comment:
please let be more strict about formatting to keep the same indentations for the same levels